brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3184 Resolve reusable block issue in Container

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3184

### Screenshots

- https://share.getcloudapp.com/o0uJKKY9

### Types of changes




- Reusable Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

## Test case 1 ##
- Drag and drop container, select any variation.
- Make container as reusable block.
- Drag and drop similar reusable block on page.
- Add some styling to reusable block
- Make sure page does not get stuck when more than 1 reusable block used on page.

## Test case 2 ##
- Wherever we have used container as child like Container as child used in Slider block.
- So drag and drop slider and design it.
- Make sure block that uses container as child should not break


### Checklist:
